### DISCUSSION TITLE
Minimal config to toggle WAL backup

### DISCUSSION THREAD

**Discussion by creckord** [zalando/postgres-operator#2376] [Q&A]
Between the Operator, Patroni, Spilo and WAL-G, I'm not sure which is the correct project to ask this, so I'm starting at the top ;)

We are currently using the Postgres Operator with logical backups to MinIO. All essential settings (MinIO coordinates, credentials, ...) are done in operator settings. This allows projects to easily opt in/out to the backups by setting `enableLogicalBackup` in their PostgreSQL resource definition.

We would now like to enable projects to switch over to WAL-G plus base backups. This should be opt-in, and we would like to avoid the need for projects to deal with the MinIO nitty gritty as far as possible. However, unlike logical backup, there does not seem to be a neat switch to toggle this backup mode.

So my question is: Which settings actually toggle WAL-G/base backups? I.e. what's the minimal amount of properties I have to leave out of the Operator config so projects can opt in by setting those?

**Comment by szelga** [comment]
I think, [this](https://opensource.zalando.com/postgres-operator/docs/reference/cluster_manifest.html#parameters-defining-how-to-clone-the-cluster-from-another-one) is what you need. the only thing I couldn't figure out is the `s3_wal_path` variable. could anyone provide an example value? even an example for AWS would be a start.
